verilog/tools/kythe/indexing_facts_tree_extractor_test.cc

```diff
@@ -422,6 +422,10 @@ TEST(FactsTreeExtractor, ModuleWithPortsNonANSIStyleTest) {
                                                         {kTag, "a"},
                                                         ", ",
                                                         {kTag, "b"},
+                                                        ", input ",
```

Comment:
I suggest we add a type to this example, like `input wire z`.  But it's good to test that both cases work.  I might even recommend that the new capability be split out into a separate test case because mixing non-ANSI and ANSI-style is technically invalid Verilog.  Verilog has this notion of implict net-type (like `wire`), but if a type is missing like in `input x`, then we just won't assign a type to it.  We should, however, have some fact about a ports direction (but that may follow-up separately).